cyto_dl/image/io/variance_reader.py

```diff
@@ -0,0 +1,61 @@
+"""ADAPTED FROM https://github.com/MMV-Lab/mmv_im2im/"""
+
+from typing import Dict, List, Sequence, Tuple, Union
+
+import numpy as np
+from aicsimageio import AICSImage
+from monai.config import PathLike
+from monai.data import ImageReader
+from monai.data.image_reader import _stack_images
+from monai.utils import ensure_tuple, require_pkg
+
+
+@require_pkg(pkg_name="aicsimageio")
+class VarianceReader(ImageReader):
+    """
+    Reader for images from Variance dataset
+    """
+    def __init__(self, channels = ['Cell', 'Nuc', 'Struct']):
+        """
+        Parameters
+        ----------
+        channels: List[str]
+            List of channels to read from Variance dataset. Options are ['Cell', 'Nuc', 'Struct', 'Brightfield']
+        """
+        super().__init__()
+
+        name_map = {'Cell': ['cmdrp'], 'Nuc':['h3342'], 'Struct': ['egfp', 'mtagrfpt'], 'Brightfield':['bright100', 'bright100x', 'tl100x', 'bright2']}
+        self.channels = []
+        for ch in channels: 
+            assert ch in ['Cell', 'Nuc', 'Struct', 'Brightfield']
+            self.channels.append(name_map[ch])
+
+
+    def read(self, data: Union[Sequence[PathLike], PathLike]):
+        filenames: Sequence[PathLike] = ensure_tuple(data)
+        img_ = []
+        for name in filenames:
+            img_.append(AICSImage(f"{name}"))
```

Comment:
for cytodata, we loaded the variance dataset using - https://github.com/aics-int/cytodl-internal-configs/blob/main/data/private/variance_3d.yaml, where images were saved as zarr files. Maybe pass in a reader argument here to handle different file types?